CurrencyConverterControl/Themes/Generic.xaml

```diff
@@ -0,0 +1,67 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="clr-namespace:CurrencyConverterControl">
+   <Style TargetType="{x:Type local:CurrencyConverterControl}">
+        <Setter Property="Background" Value="{x:Static SystemColors.WindowBrush}" />
+        <Setter Property="Foreground" Value="{x:Static SystemColors.ControlTextBrush}" />
+        <Setter Property="FontFamily" Value="{x:Static SystemFonts.CaptionFontFamily}"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type local:CurrencyConverterControl}">
+                    <Grid>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="*"/>
+                            <ColumnDefinition Width="*"/>
+                            <ColumnDefinition Width="*"/>
+                            <ColumnDefinition Width="*"/>
+                        </Grid.ColumnDefinitions>
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="22"/>
+                            <RowDefinition Height="auto"/>
+                        </Grid.RowDefinitions>
+
+                        <!--Input Value to convert-->
+                        <TextBox  x:Name="TxtInput" Grid.Column="0" Text="{Binding InputValue, RelativeSource={RelativeSource AncestorType={x:Type local:CurrencyConverterControl}},
+                            UpdateSourceTrigger=PropertyChanged , StringFormat=N2}" 
+                            HorizontalAlignment="Left" Width="130" Height="22" Background="{TemplateBinding Background}" Foreground="{TemplateBinding Foreground}" FontFamily="{TemplateBinding FontFamily}"/>
```

Comment:
in Xaml platforms Foreground, FontFamily, FontSzie properties are ambient. It means, that in most cases you should not set them directly in control parts. If they set on a parent control, they should be inherited automatically. Please try to remove them and see what happens. There are cases when it doesn't work, but it's rare and usually happens when you define UI via DataTemplates